pre-aio-node-with-zuul-ref.sh

```diff
@@ -0,0 +1,157 @@
+#!/bin/bash
+
+set -ex
+refspec="$1"
+ZUUL_CHANGES="$2"
+keep_env="$3"
+
+FETCH_URL=http://10.71.212.50/p/
+ROOT_DIR=/opt/stack/
+DEVSTACK_TOOL=/opt/stack/devstack
+NOVA_DIR=/opt/stack/nova
+TEMPEST_DIR=/opt/stack/tempest
+TMP_MERGE_BRANCH=tmp_branch_for_vgpu_patches
+DEST=${DEST:-"/opt/stack/openstack"}
+NOVA_CONF_PATH=/etc/nova
```

Comment:
Use variable for nova.conf instead of using $NOVA_CONF_PATH/nova.conf.
So suggest define NOVA_CONF_PATH=/etc/nova/nova.conf
In the future if we need customize nova-compute.conf use a new variable:
NOVA_CPU_CONF_PATH=/etc/nova/nova-compute.conf

Reply:
OK, will change it